src/main/java/org/jqassistant/contrib/plugin/php/scanner/parser/PHPFileParser.java

```diff
@@ -11,16 +11,13 @@
 import java.util.Map;
 import org.jqassistant.contrib.plugin.php.scanner.parser.helper.PHPUse;
 import org.jqassistant.contrib.plugin.php.model.PHPNamespaceDescriptor;
-import org.jqassistant.contrib.plugin.php.model.PHPFunctionDescriptor;
-import org.jqassistant.contrib.plugin.php.model.PHPTypeDescriptor;
 
 /**
  * parse php files
  * detect namespace, uses, classes and funtions
  * @author falk
  */
 public class PHPFileParser {
-    
     protected PHPFileDescriptor fileDescriptor;
     protected Helper helper;
```

Comment:
This class name gives me no idea what it is used for. Below I can see that it is used for logging. Can it be removed by a LOGGER?